src/DataMiner/SLNetTypes/Automation/AutomationTimeUpDownOptions.cs

```diff
@@ -71,14 +71,20 @@ public class AutomationTimeUpDownOptions : AutomationConfigOptions
 		/// <value>The default value for the <see cref="AutomationTimeUpDownOptions.ShowSeconds"/> property.</value>
 		public static bool DEFAULT_SHOW_SECONDS { get; protected set; }
 
-		/// <summary>
-		/// Gets or sets the number of digits to be used in order to represent the fractions of seconds.
-		/// </summary>
-		/// <value>The number of digits to be used in order to represent the fractions of seconds.</value>
-		/// <remarks>
-		/// <para>Default: 0.</para>
-		/// </remarks>
-		public int FractionalSecondsDigitsCount { get; set; }
+        /// <summary>
+        /// Gets the default value for the <see cref="AutomationTimeUpDownOptions.ShowTimeUnits"/> property.
+        /// </summary>
+        /// <value>The default value for the <see cref="AutomationTimeUpDownOptions.ShowTimeUnits"/> property.</value>
+        public static Boolean DEFAULT_SHOW_TIMEUNITS { get; protected set; }
```

Comment:
The type is not consistent with the others (e.g. see DEFAULT_SHOW_SECONDS). It however matches with what is defined in the related server commit.